css/main.css

```diff
@@ -6,6 +6,14 @@ ol li, ul li {
   margin: 10px 0;
 }
 
+ul {
+  list-style-position: inside;
+}
+
+ul ul {
+  margin-left: 40px;
```

Comment:
Have you tried `adding-left:` on the ul? This creates space that might solve what both of these are doing.

If it doesn't solve it, I'm happy to use what you got.

Reply:
If you mean on just 1st level `ul` elements, we want those to not have any left margin and also have the glyph flush against the left side of their container instead of outset. I believe what I did is the best way to accomplish that...